components/yearly_monthly_toggle/index.tsx

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useState} from 'react';
+
+import SwitchSelector from 'react-switch-selector';
+
+import {trackEvent} from 'actions/telemetry_actions';
+
+import './index.scss';
+
+interface Props {
+    updatePrice: (isMonthly: boolean) => void;
+}
+
+function YearlyMonthlyToggle({updatePrice}: Props) {
+    const [isMonthly, setIsMonthly] = useState(true);
+    const [toggleBorderClassName, setToggleBorderClassName] = useState('toggle-border');
+
+    const options = [
+        {
+            label: <p className={isMonthly ? 'text text-unselected' : 'text text-selected'}>{'Yearly'}</p>,
+            value: 'Yearly',
+        },
+        {
+            label: <p className={isMonthly ? 'text text-selected' : 'text text-unselected'}>{'Monthly'}</p>,
+            value: 'Monthly',
+        },
+    ];
+
+    const onToggleChange = () => {
+        setIsMonthly(!isMonthly);
```

Comment:
This seems odd the `not` when it's `setIsMonthly` here

Reply:
I was just trying change the isMonthly variable to the opposite of its current value. Although, I could pass in the toggle value to the `onToggleChange` function and use setIsMonthly(value === 'monthly') if that's clearer.

Reply:
I personally prefer the boolean toggle approach Emmy has here FWIW

Reply:
Didn't mean to overcomplicate this, the toggle approach LGTM